packages/ui/test/working-groups/model/getRewardAmount.test.ts

```diff
@@ -0,0 +1,13 @@
+import { getReward } from '@/working-groups/model/getReward'
+
+describe('getRewardAmount', () => {
+  it('Example use', () => {
+    const reward = getReward(1, 'forum')
+    expect(reward.value.toNumber()).toEqual(14410)
+    expect(reward.interval).toEqual(14410)
+  })
+
+  it('Not a group name', () => {
+    expect(() => getReward(1, 'incorrect')).toThrow(TypeError)
+  })
```

Comment:
I wonder if throwing will be a good idea here. Maybe a better would be `UnknownGroupReward` :thinking:

Reply:
After the second thought, we might need to have an unknown reward because it will not break Pioneer when a new group is added (but not configured). In such a scenario, we can just go with a value per 1 block instead of throwing.